.github/workflows/ci.yml

```diff
@@ -166,6 +166,22 @@ jobs:
           ${{ github.workspace }}/emulator.log
           ${{ github.workspace }}/emulator-startup.log
 
+    - name: 5. Upload coverage report
+      run: |
+        # :androidTest:connectedCheck (which depends on :androidTest:createDebugAndroidTestCoverageReport) already generated coverage report.
+
+        curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step
+
+        curl -Os https://uploader.codecov.io/latest/macos/codecov
+        curl -Os https://uploader.codecov.io/latest/macos/codecov.SHA256SUM
+        curl -Os https://uploader.codecov.io/latest/macos/codecov.SHA256SUM.sig
+
+        gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
```

Comment:
it seems to fail here, any idea?

Reply:
In the Java jobs we download the Linux signatures, but here you are downloading the Macos signatures. I suppose locally you are using a Mac, but on CI we are using Linux?

Reply:
Yes we can use that action for both the Java and Android jobs